spaces/S000123/README.md

```diff
@@ -6,7 +6,7 @@ refs:
   - doi: 10.1007/978-1-4612-6290-9
     name: Counterexamples in Topology
 ---
-Let $\{C_i\ |\ i \in \omega\}$ be a collection of disjoint dense subsets of $\mathbb{Q}$. For specificity, let \(p(i)\) be the \(i\)th prime,
+Let $\{C_i\ |\ i \in \omega\}$ be a collection of disjoint dense subsets of $\mathbb{Q}$. For specificity, let \(p(i)\) be the \((i+1)\)th prime,
 and let \(C_i\) be the rational numbers of the form $\frac{a}{p(i)^n}$, with $a$ an integer not divisible by $p(i)$.
```

Comment:
```suggestion
Let $\{C_i\ |\ i < \omega\}$ be a collection of disjoint dense subsets of $\mathbb{Q}$. For specificity, let \(\{p_i:i<\omega\}\) enumerate the prime numbers,
and let \(C_i\) be the rational numbers of the form $\frac{a}{p_i^n}$, with $a$ an integer not divisible by $p_i$.
```

Reply:
Another possible tweak to consider, avoiding the ambiguity of what the 0th prime number might be.